src/routes/accommodation.route.js

```diff
@@ -29,7 +29,14 @@ app.get('/:accommodationId/rating', valid.validateAccommodationId, BookingsContr
 app.get('/search', AccommodationsController.getAccommodationsByFilter);
 app.get('/bookmarked', checkToken, AccommodationsController.getBookmarkedAccommodations);
 app.get('/:accommodationId', AccommodationsController.getAccommodationById);
-app.post('/:accommodationId/like', checkToken, LikingsController.addLikeAccommdation);
+app.post('/:accommodationId/like', [
+  checkToken,
+  LikingsController.addLikeAccommdation
+]);
+app.get('/:accommodationId/like', [
+  checkToken,
+  LikingsController.getAccommdationLikes
+]);
```

Comment:
For consistency, avoid using the array format for middlewares.

Reply:
ok. let me fix them